libs/execution_context/ebpf_protocol.h

```diff
@@ -390,6 +391,14 @@ typedef struct _ebpf_operation_ring_buffer_map_async_query_reply
     ebpf_ring_buffer_map_async_query_result_t async_query_result;
 } ebpf_operation_ring_buffer_map_async_query_reply_t;
 
+typedef struct _ebpf_operation_ring_buffer_map_write_data_request
+{
+    struct _ebpf_operation_header header;
+    ebpf_handle_t map_handle;
+    size_t data_length;
```

Comment:
Why is this needed? Header already contains total request length, so this is redundant.